tests/accelerators/test_hpu.py

```diff
@@ -240,3 +240,31 @@ def test_hpu_auto_device_count():
 def test_hpu_unsupported_device_type():
     with pytest.raises(MisconfigurationException, match="`devices` for `HPUAccelerator` must be int, string or None."):
         Trainer(accelerator="hpu", devices=[1])
+
+
+@RunIf(hpu=True)
+def test_strategy_params_with_hpu_parallel_strategy():
+    _bucket_cap_mb = 100
+    _gradient_as_bucket_view = True
+    _static_graph = True
+    _find_unused_parameters = True
+    trainer = Trainer(
```

Comment:
I don't think this test needs to instantiate a `Trainer`, you can do

`strategy = HPUParallelStrategy(...)`

and use that variable for the assertions